src/org/opencraft/server/model/Store.java

```diff
@@ -50,21 +50,27 @@
 public class Store {
   private HashMap<String, StoreItem> items = new HashMap<String, StoreItem>(16);
 
+  public int bigTNTPrice = 70;
```

Comment:
Maybe make these static constants so they can be referenced outside easily also because they currenlty cannot be changed during runtime. If the intention was to make them change during runtime, then each store item will also need to be refactored to support that then.

Reply:
Should be good to go now. I thought I did make the prices static constants but I guess not.